torchinfo/formatting.py

```diff
@@ -118,12 +122,13 @@ def layer_info_to_row(self, layer_info: LayerInfo, reached_max_depth: bool) -> s
                 new_line += self.format_row(f"{prefix}{inner_name}", inner_layer_info)
         return new_line
 
-    def layers_to_str(self, summary_list: list[LayerInfo]) -> str:
+    def layers_to_str(self, layers_summary: Any) -> str:
```

Comment:
No need to change this. We shouldn't pass total_params as a parameter. Just use it in self.total_params

Reply:
Don't pass the total_params as parameter? We should add total_params to summary_list? Or some other ideas? I really don't think the current one is a good implementation.

Reply:
Oh I didn't realize total_params was not on this class. Ignore this comment